CMSIS/DSP/Include/arm_math.h

```diff
@@ -306,6 +306,9 @@
   #define ARM_MATH_CM0_FAMILY            1
 #elif defined (__ARM_ARCH_8M_MAIN__)
 //#define ARM_MATH_CM0_FAMILY            0
+#elif defined (ARM_MATH_NEON)
```

Comment:
This preprocessor condition does not fit the others. I don't expect ARM_MATH_NEON to be a predefined setting set by the compiler. Can we have this in a generic way, i.e. automatically deduced?

Reply:
We will switch to a dsp_config.h in a future commit.

Reply:
Please add this pair of strings as placeholder for use
DSP librady with Cortex-A9:
_#elif defined (__ARM_ARCH_7A__)
//#define ARM_MATH_CM0_FAMILY            0_

For NEON detecting wee can use __ARM_NEON symbol